app/uk.gov.hmrc.helptosavestridefrontend/views/application_cancelled.scala.html

```diff
@@ -15,18 +15,33 @@
  *@
 
 @import uk.gov.hmrc.helptosavestridefrontend.controllers.routes
+@import uk.gov.hmrc.helptosavestridefrontend.config.FrontendAppConfig
+@import uk.gov.hmrc.helptosavestridefrontend.views.html.layout
+@import uk.gov.hmrc.helptosavestridefrontend.views.html.partials._
 
-@this(main: main_template)
+@this(
+  layout: layout,
+  h1: h1,
+  a: link
+)
 
-@()(implicit messages: Messages)
+@()(
+  implicit request: Request[_],
+  config: FrontendAppConfig,
+  messages: Messages
+)
 
-@title = @{Messages("hts.application.cancelled.title.h1")}
+@layout(title = Some(messages("hts.application.cancelled.title.h1"))) {
 
-@main(
-  title = title,
-  bodyClassesOption = None) {
+  @h1(messages("hts.application.cancelled.title.h1"))
 
-  <h1 class="heading-large">@title</h1>
-
-  <a role="button" class="button" href=@routes.StrideController.getEligibilityPage().url>@messages("hts.global.end-call.button")</a>
+  @a(
```

Comment:
links styled as buttons (as per the original template) should be rendered using the GovukButton component and passing an href value